scarb/src/manifest_editor/add.rs

```diff
@@ -136,6 +136,17 @@ impl Dep {
                     .unwrap_or(git),
                 Err(_) => git,
             };
+            let git_url = Url::parse(&git)
```

Comment:
We do not actually need to add anything here. 
Please note, that we already call `Url::parse` at `133`. We only need to stop ignoring the errors it produces.
As of now we do `Err(_) => git`, but we could just drop whole match, and do `Url::parse(&git)?` instead, to bubble the error up.